tests/end2end/features/downloads.feature

```diff
@@ -684,3 +684,16 @@ Feature: Downloading things from a website.
         When I set downloads.location.prompt to false
         And I open 500-inline
         Then the error "Download error: *INTERNAL SERVER ERROR" should be shown
+
+    ## External download path fileselector
+
+    Scenario: Select download path
+        When I set downloads.location.prompt to true
+        And I set fileselect.handler to external
+        And I setup a fake folder fileselector selecting "tests/end2end/data/backforward/" and writes to a temporary file
+        And I open data/downloads/downloads.html
+        And I run :click-element id download
+        And I wait for the download prompt for "*/download.bin"
```

Comment:
This doesn't work because the tests end up waiting for:

```
----> Waiting for "Asking question <qutebrowser.utils.usertypes.Question default='*/download.bin' mode=<PromptMode.do… in the log
```

but the log output reads:

```
11:25:14.821 DEBUG    prompt     prompt:ask_question:197 Starting loop.exec() for <qutebrowser.utils.usertypes.Question default='/tmp/pytest-of-florian/pytest-1/test_select_download_path0/downloads/' mode=<PromptMode.download: 5> option=None text='Please enter a location for <b>http://localhost:59185/data/downloads/download.bin</b>' title='Save file to:'>
```

note the `default='*/download.bin'` vs. `default='/tmp/pytest-of-florian/pytest-1/test_select_download_path0/downloads/' - in other words, matching the filename here requires `And I set downloads.location.suggestion to both`.

I'd just do

```suggestion
        And I wait for the download prompt for "*"
```

like in some of the other tests, since the filename doesn't matter for what we actually want to test.